index.js

```diff
@@ -1,8 +1,15 @@
-function pipeLogsToTerminal() {
+function pipeLogsToTerminal(config) {
   let oldConsoleWarn;
   let oldConsoleError;
   let logs = [];
 
+  Cypress.on('fail', error => {
+    const [type, message] = logs[logs.length - 1];
+    logs[logs.length - 1] = [type, message, 'failed'];
```

Comment:
I don't think so. 

Previously, every last log was marked as an error. But here the log is marked as error only when the `Cypress error` event is triggered. Of course, I am assuming the last command cause the error. 

What do you think?

Reply:
Well we had the issue that the last command is always considered the failed one, for the we introduced the tracking and marking through the command status type. But with this change we reintroduced marking the last one as the failing. The issue was with some commands being executed after the real last failing on in certain cases, and this reintroduces that issue no ?

Reply:
I have taught a little more about this and actually it seems fine: since when the error event is emitted at the exact point when the last command fails we most likely won't have any new logs in the list so marking the last one as failing this way seems better. Although I still have some doubt. But let's proceed like this and see if we get any additional issues.